packages/service-clients/end-to-end-tests/azure-client/src/test/audience.spec.ts

```diff
@@ -4,355 +4,461 @@
  */
 
 import { strict as assert } from "node:assert";
+import { AxiosResponse } from "axios";
 
-import { AzureClient } from "@fluidframework/azure-client";
-import { ScopeType } from "@fluidframework/azure-client/internal";
+import { AzureClient, type AzureContainerServices } from "@fluidframework/azure-client";
 import { AttachState } from "@fluidframework/container-definitions";
 import { ConnectionState } from "@fluidframework/container-loader";
-import { ContainerSchema } from "@fluidframework/fluid-static";
+import { ContainerSchema, type IFluidContainer } from "@fluidframework/fluid-static";
+import { ScopeType } from "@fluidframework/azure-client/internal";
 import { SharedMap } from "@fluidframework/map/internal";
 import { timeoutPromise } from "@fluidframework/test-utils/internal";
 
-import { createAzureClient } from "./AzureClientFactory.js";
+import { createAzureClient, createContainerFromPayload } from "./AzureClientFactory.js";
 import { configProvider, waitForMember } from "./utils.js";
+import * as ephemeralSummaryTrees from "./ephemeralSummaryTrees.js";
+
+const testMatrix = [
+	{
+		variant: "Durable Container",
+		options: {
+			isEphemeral: false,
+		},
+	},
+	{
+		variant: "Ephemeral Container",
+		options: {
+			isEphemeral: true,
+		},
+	},
+];
+
+for (const testOpts of testMatrix) {
+	describe(`Fluid audience (${testOpts.variant})`, () => {
+		const connectTimeoutMs = 10_000;
+		let client: AzureClient;
+		let schema: ContainerSchema;
+		const isEphemeral: boolean = testOpts.options.isEphemeral;
```

Comment:
How do we detect when the tests are running in tinylicious? Maybe it could even be setup to where the "isEphemeral: true" item of the test matrix is removed altogether for t9s instead of skipped; this would make the E2E test logs quicker and less cluttered for t9s

Reply:
The `useAzure` line above is how to know. In other words, `process.env.FLUID_CLIENT !== "azure"`.

Yeah, removing it from the test matrix would also work! Let's see if client folks have an opinion on whether to skip Ephemeral matrix for T9s or not

Reply:
Added change for this :)